mohilevets/lab1-BasicIO/Makefile

```diff
@@ -0,0 +1,2 @@
+lab1:
+	gcc  main.c user.c user.h  -o lab1
```

Comment:
1. What `gcc` would be used, seems that system-wide.
    But XC32 gcc toolchain uses `xc32-`prefix:
    ```
    [thd@aspire bin]$ ls
    bin                xc32-ar       xc32-collect2     xc32-ld       xc32-strings
    device_files       xc32-as       xc32-conv         xc32-nm       xc32-strip
   deviceSupport.xml  xc32-bin2hex  xc32-cpp          xc32-objcopy  xclm
   manifest           xc32-c++      xc32_device.info  xc32-objdump
   roam.lic           xc32-cc1      xc32-elfedit      xc32-ranlib
   verifyinst         xc32-cc1plus  xc32-g++          xc32-readelf
   xc32-addr2line     xc32-c++filt  xc32-gcc          xc32-size
   ```
   So this is either build with GNU GCC or won't build at all
   Don't forget that XC GCC toolchain could be contained in different dirs depending on installation dir

2. If You decide to create Makefile yourself, be aware this won't build because in command user-headers aren't passed correctly and MCU-related headers aren't linked at all

3. All sources should end with newline (`\n`) symbol